eng/Version.Details.xml

```diff
@@ -87,10 +91,6 @@
       <Sha>b57dee7cec971021547a7b8a36a46d7271fea99e</Sha>
       <SourceBuild RepoName="fsharp" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="dotnet-format" Version="9.0.512801">
-      <Uri>https://github.com/dotnet/format</Uri>
-      <Sha>de0824fccb49f4c23a9a55c6d35b1f1fb23bcfa5</Sha>
-    </Dependency>
     <!-- Intermediate is necessary for source build. -->
     <Dependency Name="Microsoft.SourceBuild.Intermediate.format" Version="9.0.512801">
       <Uri>https://github.com/dotnet/format</Uri>
```

Comment:
I wonder if this one is still necessary, but would have to ask a source build expert.

Reply:
On second look, this should probably be removed. It says it is building out of the format repo but we shouldn't be relying on anything from that repo.

Reply:
Oh I forget to push the change

Reply:
Hi @dotnet/source-build-internal  could I get your knowledge about how to change this file? 
I am moving dotnet/format to dotnet/sdk repo, could this be removed?